src/dataDisplay/personPanel/__test__/personPanel.test.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * To run this test from the church-management/client folder, execute the following command:
+ * npx jest ./src/dataDisplay/personPanel/__test__/personPanel.test.js
+ */
+import { describe, expect, it } from '@jest/globals';
+import { mount } from 'enzyme';
+import React from 'react';
+import PersonPanel from '../personPanel';
+import PersonPanelDetails from '../personPanelDetails';
+import MockedTheme from '../../../testUtils/mockedTheme';
+import PersonPanelSummary from '../personPanelSummary';
+
+describe('<PersonPanel />', () => {
+    const minimalChildren = [
+        <PersonPanelSummary
+            key="summary"
+        >
+            Summary
+        </PersonPanelSummary>,
+        <PersonPanelDetails
+            key="details"
+        >
+            Details
+        </PersonPanelDetails>,
+    ];
+    const props = {
+        id: 'block_name--element_name-modifier',
+        isExpanded: false,
+        onChange: jest.fn(),
+    };
+
+    it('Should render without problems', () => {
+        const wrapper = mount(
```

Comment:
Could we wrap mounting wrapper on function `beforeEach`. Like below
```
beforeEach(() => {
    const wrapper = mount(
            <MockedTheme>
                <PersonPanel
                    {...props}
                >
                    {minimalChildren}
                </PersonPanel>
            </MockedTheme>,
        );
  ));
```
Like this, we don't need to iterate over other checks

Reply:
Are we using exactly the same props in every test case?

Reply:
I have one or two tests that are messing with props and defined the `mount/shallow` in each test case because of that. Let me know if this is blocking.